sys-apps/nawk/nawk-20121220-r3.ebuild

```diff
@@ -1,13 +1,13 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 inherit toolchain-funcs
 
 DESCRIPTION="Brian Kernighan's pattern scanning and processing language"
-HOMEPAGE="http://cm.bell-labs.com/cm/cs/awkbook/index.html"
-SRC_URI="mirror://gentoo/${P}.tar.gz"
+HOMEPAGE="https://www.cs.princeton.edu/~bwk/btl.mirror/"
+SRC_URI="https://www.cs.princeton.edu/~bwk/btl.mirror/awk.tar.gz -> ${P}.tar.gz"
```

Comment:
No that doesn't work either, as upstream unpredictably changes the content of the tarball and then if would be a different version.

Reply:
Hmm.. yes i see.
Well, i guess my options are now to either i ask upstream to provide a versioned package (which, considering the state of the package, won't happen) or stay with the gentoo mirror for the source?

Reply:
It never hurt to ask though...

I also email them that patch https://github.com/junghans/nawk/commit/1ab0265fec229c5e15d2f252360eca1834c679c7 at some point.

Reply:
Sure, I'll send a mail and ask for a versioned package.
Can we keep the PR open in the mean time?